### DISCUSSION TITLE
Does Testcontainers work with Podman instead of Docker?

### DISCUSSION THREAD

**Discussion by jorgegarciarey** [testcontainers/testcontainers-java#7419] [General]
Hi team,

I've tried to create a post via Slack but it seems that I have not permissions, so, I type here:

At our company, we'd like to use Podman to work instead of Docker, but, we have several doubts if TestContainers could work with this tool. I was reviewing Slack with `podman` topics, but it's not clear to me if Podman would work or not.

If you need something, let me know, please

Cheers.

**Comment by eddumelendez** [comment]
Hi @jorgegarciarey, sorry for the delay. Yes, Podman works with Testcontainers. See [the documentation](https://java.testcontainers.org/supported_docker_environment/#podman) and follow the steps to run Testcontainers using Podman.

**Reply by rgala** [reply]
What about Windows support?
